docs/index.md

```diff
@@ -39,19 +39,48 @@ terraform {
   required_providers {
     juju = {
       version = "~> 0.3.1"
-      source  = "juju/juju"
+      source  = "canonical/juju"
```

Comment:
Was this intentional? The terraform registry path right now is still `juju/juju`?